GameOfLife/Program.fs

```diff
@@ -0,0 +1,67 @@
+module GameOfLife.Program
+open System
+open System.Security.Cryptography
+open System.Text
+open System.Threading
+
+let rows = 15
+let columns = 15
+let timeout = 500
+let mutable runSimulation = true
+
+type Status = ``💀`` = 0 | ``😁`` = 1
+
+let private nextGeneration (currentGrid: Status [,]) = 
+    let nextGeneration = Array2D.zeroCreate rows columns
+    // Loop through every cell 
+    for row in 1..(rows-2) do
+        for column in 1..(columns-2) do
+            // find your alive neighbor
+            let mutable ``😁Neighbors`` = 0
+            for i in -1..1 do
+                for j in -1..1 do
+                    ``😁Neighbors`` <- ``😁Neighbors`` + int currentGrid.[row+i, column+i]
+            let currentCell = currentGrid.[row, column]
+            // The cell needs to be subtracted 
+            // from its neighbours as it was  
+            // counted before 
+            ``😁Neighbors`` <- ``😁Neighbors`` - int currentCell
+            // Implementing the Rules of Life 
+            nextGeneration.[row,column] <-
+                match currentCell with
+                // Cell is lonely and dies OR Cell dies due to over population
+                | Status.``😁`` when ``😁Neighbors`` < 2 || ``😁Neighbors`` > 3 -> Status.``💀``
+                // A new cell is born 
+                | Status.``💀`` when ``😁Neighbors`` = 3 -> Status.``😁``
+                // stays the same
+                | _ -> currentCell
```

Comment:
TBH, this is a Joke PR, and I wasn't really under the opinion that enumoji's are a good idea. So I used an emoji in this neighbors variable, because that's how alive is referenced in the Enum, and it was funnier that way.  However, on review, if emoji's were only used in the enums, it would really makes the match case pop, so maybe they are some pro's to enumoji's 🤔.
```suggestion
            let mutable aliveNeighbors = 0
            for i in -1..1 do
                for j in -1..1 do
                    aliveNeighbors <- aliveNeighbors + int currentGrid.[row+i, column+i]
            let currentCell = currentGrid.[row, column]
            // The cell needs to be subtracted 
            // from its neighbours as it was  
            // counted before 
            aliveNeighbors <- aliveNeighbors - int currentCell
            // Implementing the Rules of Life 
            nextGeneration.[row,column] <-
                match currentCell with
                // Cell is lonely and dies OR Cell dies due to over population
                | Status.``😁`` when aliveNeighbors < 2 || aliveNeighbors > 3 -> Status.``💀``
                // A new cell is born 
                | Status.``💀`` when aliveNeighbors = 3 -> Status.``😁``
                // stays the same
                | _ -> currentCell
```